src/jobservice/pool/redis_pool.go

```diff
@@ -130,8 +135,18 @@ func (gcwp *GoCraftWorkPool) Start() {
 			return
 		}
 
+		startTimes := 0
+	START_MSG_SERVER:
```

Comment:
Can we use a for loop？

Reply:
Just preference, label and goto is more simple